repository_service_tuf/cli/artifact/add.py

```diff
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: MIT
+
+from click import Context
+
+from repository_service_tuf.cli import click, console
+from repository_service_tuf.cli.artifact import artifact
+from repository_service_tuf.helpers.api_client import URL, send_payload
+from repository_service_tuf.helpers.cli import (
+    create_artifact_payload_from_filepath,
+    validate_cli_add_artifact_options,
+)
+
+
+@artifact.command()
+@click.argument(
+    "filepath",
+    type=click.Path(exists=True),
+    # Currently, this is required. If we support adding artifacts without
+    # giving the filepath then we need to set it to `False` and implement our
+    # own validation to check whether the argument is needed based on the
+    # passed user options.
+    required=True,
+)
+@click.option(
+    "-p",
+    "--path",
+    help="A custom path (`TARGETPATH`) for the file, defined in the metadata.",
+    type=str,
+    required=True,
+    default=None,
+)
+@click.pass_context
+def add(context: Context, filepath: str, path: str) -> str:
+    """
+    Add artifacts to the metadata (`targets.json`).
```

Comment:
The artifact will be added to the hash bin delegated role, in the future it could be another custom metadata as well.
It is better to remove it IMO.
```suggestion
    Add artifacts to the TUF metadata.
```

Reply:
I agree, good point